tools/test-framework/src/types/single/node.rs

```diff
@@ -138,7 +138,7 @@ impl FullNode {
             default_gas: None,
             max_gas: Some(3000000),
             gas_adjustment: None,
-            gas_multiplier: None,
+            gas_multiplier: Some(1.1),
```

Comment:
This needs to be set to 1.1 for the integration tests on Osmosis and Wasmd to pass. Should there be adjustment on the default values so that there is no need for users to configure such magic value depending on the chains?

Reply:
Yes, let's change it to 1.1 in a different issue/PR since that seems to work in most if not all cases.